var/spack/repos/builtin/packages/miopengemm/package.py

```diff
@@ -11,15 +11,24 @@ class Miopengemm(CMakePackage):
     and kernel generator"""
 
     homepage = "https://github.com/ROCmSoftwarePlatform/MIOpenGEMM"
-    url      = "https://github.com/ROCmSoftwarePlatform/MIOpenGEMM/archive/1.1.6.tar.gz"
```

Comment:
You should keep a url here (and update it) so that `spack versions` and `spack checksum` continue to work.